src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    public String getDescription(){
+        return description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Maybe you want to store the `tick` or `X` symbol in a constant such as
`public static final TICK_SYMBOL = "\u2713"`
`public static final X_SYMBOL = "\u2718"`